lms/djangoapps/courseware/tests/test_credit_requirements.py

```diff
@@ -0,0 +1,138 @@
+"""
+Tests for credit requirement display on the progress page.
+"""
+
+import datetime
+
+from mock import patch
+from pytz import UTC
+
+from django.conf import settings
+from django.core.urlresolvers import reverse
+
+from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
+from xmodule.modulestore.tests.factories import CourseFactory
+from student.tests.factories import UserFactory, CourseEnrollmentFactory
+from util.date_utils import get_time_display, DEFAULT_SHORT_DATE_FORMAT
+
+from openedx.core.djangoapps.credit import api as credit_api
+from openedx.core.djangoapps.credit.models import CreditCourse
```

Comment:
Minor: these imports don't follow the order recommended by PEP 8 (i.e., standard library, third-party, first-party).